data/blog/greenart7c3-receives-lts-grant.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Long-Term Support For Greenart7c3"
+date: '2024-10-16'
+tags: ['OpenSats', 'grants', 'nostr', 'LTS']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/58-greenart7c3-lts.jpg']
+summary: "OpenSats Welcomes Greenart7c3 as an LTS Grantee."
+---
+
+OpenSats is pleased to announce long-term support for
+[Greenart7c3](https://github.com/greenart7c3) for his ongoing contributions to
+the nostr ecosystem.
+
+Greenart7c3 is the creator and lead developer of
+[Citrine](https://github.com/greenart7c3/Citrine), an Android-based nostr relay,
+and [Amber](https://github.com/greenart7c3/Amber), an Android-based nostr signer
+using [NIP-46](https://github.com/nostr-protocol/nips/blob/master/46.md) to
+enhance private key security practices for users utilizing the same key across
+multiple clients.
+
+Citrine functions as both a local relay and a shared database, allowing users to
+send and receive events while maintaining control of their data offline. Amber
+focuses on enhancing security by keeping sensitive keys in a single, secure app
+that is responsible for remotely signing events on behalf of multiple clients.
+These developments by Greenart7c3 contribute to the broader goal of
+decentralizing and safeguarding user privacy within the nostr ecosystem.
+
+With support from this LTS grant, Greenart7c3 will focus on the continued
+development of Citrine and Amber, with plans including:
+
+- Extending Amber and Citrine to cross-platform usability beyond Android
+  devices.
+
+- Integrating Citrine into [Amethyst](/blog/vitor-pamplona-receives-lts-grant)
+  to enhance its user interface and functionality.
```

Comment:
```suggestion
- Improve the user experience when using Amber and Citrine in [Amethyst](/blog/vitor-pamplona-receives-lts-grant).
```
I think this way would be better since vitor doesn't want a database in amethyst yet